soot-infoflow-android/src/soot/jimple/infoflow/android/iccta/MessageHandler.java

```diff
@@ -28,7 +28,7 @@ public Set<SootClass> getAllHandlers() {
 			for (Iterator<SootClass> iter = Scene.v().getApplicationClasses().snapshotIterator(); iter.hasNext();) {
 				SootClass sootClass = iter.next();
 
-				SootClass tmpClass = sootClass.getSuperclassUnsafe();
+				SootClass tmpClass =  Scene.v().getSootClass(sootClass.getName());
```

Comment:
This is equal to ``tmpClass = sootClass``. The original code obtained the superclass. Please check,